package.json

```diff
@@ -26,9 +26,10 @@
     "build": "yarn cmd build",
     "watch": "yarn cmd watch",
     "package": "yarn cmd package",
-    "test": "cross-env NATIVE_EXTENSIONS_DIR=build/native-extensions cross-env jest -i --detectOpenHandles -c jest.config.js",
```

Comment:
Personally, I prefer forcing the user to choose between unit, integration, and E2E instead of having a generic script that runs one of them. 

But I also, agree with you that we could have a test script that, by default, runs the unit tests, I just find it redundant TBH